contributors/guide/github-workflow.md

```diff
@@ -7,38 +7,41 @@ description: |
   sync with upstream and commit hygiene.
 ---
 
+This document is an overview of the GitHub workflow used by the
+Kubernetes project. It includes tips and suggestions on keeping your
+local environment in sync with upstream and how to maintain good
+commit hygiene.
+
 ![Git workflow](git_workflow.png)
 
-### 1 Fork in the cloud
+## 1 Fork in the cloud
```

Comment:
Curious, if the guidelines say anything about using Steps as header? `1. Fork in the cloud` is more readable instead of `1 Fork in the cloud` (same with other steps as header), and we may want to list steps that way (or remove numbers or something similar) instead of how it's currently. (I took a quick look at the guidelines but couldn't get answer.)